Makefile

```diff
@@ -106,6 +107,19 @@ benchmark-gap:
 	-c ${KERNMLOPS_CONFIG_FILE} \
 	--benchmark gap
 
+benchmark-mongodb:
+	@mongod --dbpath "$(YCSB_BENCHMARK_DIR)/mongo_db" --fork --logpath /var/log/mongodb.log || { echo "Error is expected, just means that the server was already running"; true; }
+	@python python/kernmlops collect -v \
+		-c ${KERNMLOPS_CONFIG_FILE} \
+		--benchmark mongodb
+
+load-mongodb:
+	@echo "Loading MongoDB benchmark"
+	@python $(YCSB_BENCHMARK_DIR)/ycsb-0.17.0/bin/ycsb load mongodb -s \
+		-P "$(YCSB_BENCHMARK_DIR)/ycsb-0.17.0/workloads/workloada" \
+		-p recordcount=1000000 \
+		-p mongodb.url=mongodb://localhost:27017/ycsb
+
```

Comment:
please add a new-directive here:

```
start-mongodb:
  @mkdir -p "$(YCSB_BENCHMARK_DIR)/mongo_db"
  @mongod --dbpath "$(YCSB_BENCHMARK_DIR)/mongo_db" --fork --logpath /var/log/mongodb.log || { echo "Error is expected, just means that the server was already running"; true; }
```

and then call it instead inside `benchmark-mongodb` and `load-mongodb`
via `@${MAKE} start-mongodb`

this will make the process more reliable